### ISSUE TITLE
Access Rserve settings values from within R /Java?

### ISSUE THREAD

**Issue by wiwie** [s-u/Rserve#89]
This is more of a question than anything else:

Is it possible, to retrieve the settings of the running Rserve during runtime from within R or directly in Java?

I found the --RS-settings parameter. However, I am not sure, how this could be used from within R or Java.

**Comment by pzingg** [comment]
I have a similar request, namely for my R code to be able to detect that it is running on Rserve, and therefore to handle exiting in a different way. My temporary workaround is to use Sys.setenv at startup to set an environment variable that my R code can detect at runtime, by using the “source” or “eval” configuration options.

**Comment by s-u** [comment]
No, there is currently no way to retrieve the Rserve settings, they are entirely internal at this point. It is a fair request, though.
